BlueprintUI/Sources/Layout/Stack.swift

```diff
@@ -180,6 +180,37 @@ extension StackElement {
             child: child
         )
     }
+
+    /// Adds a `Spacer` element to the stack, which will grow (and shrink) depending on the amount
+    /// of left over available space when measuring the elements.
+    public mutating func addFlexibleSpacer() {
+        precondition(
```

Comment:
This doesn't seem worth crashing for IMO. I'd expect a flexible spacer to just not space when in an underflow distribution mode that doesn't cause this to do more than the minimum size (currently `1`).

Curious what others think.

Reply:
I guess I did the crash over a silent / confusing failure that people usually run into, since a lot of the time people don't realize that the flexible sizing has no effect outside of these underflow options.